core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -851,30 +851,32 @@ private void locationChanged(Item item, String oldName, String newName) {
     /**
      * Time when the fingerprint has been captured.
      */
-    private final @Nonnull Date timestamp;
+    public final @Nonnull Date timestamp;
```

Comment:
Please rather get used to setters./getters in the code. Public fields are nightmare in Java when it comes to maintaining binary compatibility

Reply:
@oleg-nenashev Got it :) Made the requested changes!